### ISSUE TITLE
Problema con instalación php

### ISSUE THREAD

**Issue by ssepul21** [IIC2413/Syllabus-2024-2#22]
Hola!
 Espero se encuentren bien. He tenido problemas para instalar php en mi pc.
 Pude seguir los pasos del [vídeo tutorial de instalación](https://youtu.be/AOQgYhHg0J8) hasta el minuto 2:05, pero cuando hago `php --version` en la consola, me aparece como si no hubiese instalado nada. Adjunto una foto con lo que sale.
 ¿Lo descargado debe estar en una carpeta en concreto dentro del almacenamiento? No se me ocurre que puede ser en concreto 😔

![image](https://github.com/user-attachments/assets/d6cf2f98-3644-444c-846a-f7ec06903cfd)

 Gracias de antemano!

**Comment by TomasKetterer** [comment]
Hola! Me puedes dar un poco más de información al respecto? La carpeta php-8.3.10 la traspasaste al disco (C:)?

**Comment by ssepul21** [comment]
Claro! Si, pasé esa carpeta al disco. No lo dejé dentro de ninguna otra carpeta

**Comment by TomasKetterer** [comment]
El paso donde tienes que determinar el path en el entorno de variables, lo hiciste sobre el usuario tuyo o en el sistema? (En caso de que tengas varios usuarios, asegurate de que el path quede justo en el usuario que estás ocupando)
